docs/adding-a-new-app.md

```diff
@@ -1,248 +1,178 @@
 ) {
-  govuk_postgresql::db { 'my_app_production':
-    user                    => 'my_app',
+  govuk_postgresql::db { 'myapp_production':
+    user                    => 'myapp',
     password                => $password,
     allow_auth_from_backend => true,
     backend_ip_range        => $backend_ip_range,
+    rds                     => $rds,
   }
 }
 ```
 
-Then add the database config to the main application manifest.
-
-```
-$db_hostname = undef,
-$db_username = 'my_app',
-$db_password = undef,
-$db_name = 'my_app_production',
-.
-.
-.
-.
-if $::govuk_node_class !~ /^development$/ {
- govuk::app::envvar::database_url { $app_name:
-   type     => 'postgresql',
-   username => $db_username,
-   password => $db_password,
-   host     => $db_hostname,
-   database => $db_name,
- }
-}
-```
-
-Add a value for `db_hostname` and `db::backend_ip_range` to the `hieradata/common.yaml` file.
+Start with the following configuration for the app module variables.
 
 ```
 # hieradata/common.yaml
-govuk::apps::content_performance_manager::db_hostname: "postgresql-primary-1.backend"
-govuk::apps::content_performance_manager::db::backend_ip_range: "%{hiera('environment_ip_prefix')}.3.0/24"
-```
-
-Add the application to the database servers. For example, if your application has a
-postgresql database, add an entry to the `govuk::node::s_postgresql_base` class in `modules/govuk/manifests/node/s_postgresql_base.pp`
-
-```
-# modules/govuk/manifests/node/s_postgresql_base.pp
-class govuk::node::s_postgresql_base inherits govuk::node::s_base {
-  include govuk::apps::my_app::db
-  .
-  .
-  .
-}
-```
-
-Add a value for `db_password` in each of the credential files in the `govuk-secrets` repo.
-This entry should be encrypted.
-
-Add a values for `db::password` and `db_password` in `hieradata/vagrant_credentials.yaml`.
-This is necessary for puppet specs to pass on CI.
-
-### AWS RDS Database Management with db_admin
-
-`db_admin` needs to know about the database in order to administrate it. See docs in `govuk-aws` on [RDS Database Management](https://github.com/alphagov/govuk-aws/blob/master/doc/guides/rds-database-management.md).
-
-For example when using PostgreSQL, to enable `db_admin` edit [modules/govuk/manifests/node/s_db_admin.pp](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_db_admin.pp) and add `your_app`:
-
-```
-# include all PostgreSQL classes that create databases and users
-  class { '::govuk::apps::your_app::db': } ->
-  class { '::govuk::apps::another_app::db': } ->
-  class { '::govuk::apps::yet_another_app::db': }
-```
-
-## Environment variables
+govuk::apps::myapp::db_hostname: "postgresql-primary-1.backend"
+govuk::apps::myapp::db::backend_ip_range: "%{hiera('environment_ip_prefix')}.3.0/24"
 
-If your application needs other environment variables, they will need to be added to
-the application manifest too.
+# hieradata_aws/common.yaml
+govuk::apps::myapp::db_hostname: "postgresql-primary-1.backend"
```

Comment:
Small nitpick but that hostname is carrenza specific if I remember correctly and we have a different one for AWS.